packages/apidom-ns-openapi-2/src/refractor/visitors/open-api-2/responses/DefaultVisitor.ts

```diff
@@ -1,34 +1,38 @@
-import stampit from 'stampit';
+import { Mixin } from 'ts-mixer';
 import { T as stubTrue } from 'ramda';
 import { ObjectElement } from '@swagger-api/apidom-core';
 
 import { isReferenceLikeElement } from '../../../predicates';
 import { isReferenceElement, isResponseElement } from '../../../../predicates';
-import AlternatingVisitor from '../../generics/AlternatingVisitor';
+import AlternatingVisitor, {
+  AlternatingVisitorOptions,
+  Alternator,
+} from '../../generics/AlternatingVisitor';
 import FallbackVisitor from '../../FallbackVisitor';
 
-const DefaultVisitor = stampit(AlternatingVisitor, FallbackVisitor, {
-  props: {
-    alternator: [
+class DefaultVisitor extends Mixin(AlternatingVisitor, FallbackVisitor) {
+  public declare readonly alternator: Alternator[];
```

Comment:
AlternatingVisitor.prototype.alternator is already defined as `protected readonly alternator: Alternator[];` so redeclaring here shoudn't be needed?

Reply:
Without redeclaring it, the assignment to `alternator` in the constructor gets us this error: `Cannot assign to 'alternator' because it is a read-only property.`

Reply:
We can make it `-readonly` in AlternatingVisitor, the same as we do for properties in FixedFieldsVisitor.